scripts/lib/mkcloud-libvirt.sh

```diff
@@ -319,18 +321,20 @@ function libvirt_do_onhost_deploy_image()
     echo "Cloning $role node vdisk from $image ..."
     safely qemu-img convert -t none -O raw -S 0 -p $cachedir/$image $disk
 
-    # resize the last partition only if it has id 83
-    local last_part=$(fdisk -l $disk | grep -c "^$disk")
-    if fdisk -l $disk | grep -q "$last_part *\* *.*83 *Linux" ; then
-        echo -e "d\n$last_part\nn\np\n$last_part\n\n\na\n$last_part\nw" | fdisk $disk
-        local part=$(kpartx -asv $disk|perl -ne 'm/add map (\S+'"$last_part"') / && print $1')
-        test -n "$part" || complain 31 "failed to find partition #$last_part"
-        local bdev=/dev/mapper/$part
-        safely fsck -y -f $bdev
-        safely resize2fs $bdev
-        time udevadm settle
-        sleep 1 # time for dev to become unused
-        safely kpartx -dsv $disk
+    if is_suse; then
+        # resize the last partition only if it has id 83
+        local last_part=$(fdisk -l $disk | grep -c "^$disk")
```

Comment:
this also is not SUSE-specific, if you want to support to skip this, rather add something like 
`if [[ ! $skip_partition_resize ]]`